### PR TITLE
Release 2.3.1

### PR DESCRIPTION
**Pull request by rjbou**

### CODE REVIEW

File: CHANGES
Comment (kit-ty-kate) [inline CHANGES:3]:
```suggestion
* Use opam download command mechanism to select a specific download command [#159 @Emilios1995]
```

**Review by kit-ty-kate — APPROVED** [review]
otherwise lgtm

**Comment by rjbou** [comment]
CI will broke again as packages are not yet published. Once fixed, good for merge & release

**Comment by kit-ty-kate** [comment]
I don't think we should wait for the opam PR to be merged upstream, opam-repo-ci will check it anyway.

**Pull request merged**
